lib/Raven/Client.php

```diff
@@ -1435,4 +1435,205 @@ public function close_curl_resource()
             $this->_curl_instance = null;
         }
     }
+
+    /**
+     * Interpolates context values into the message placeholders.
+     *
+     * @param object|string $message
+     * @param array         $context
+     *
+     * @return string
+     * @doc http://www.php-fig.org/psr/psr-3/
+     */
+    public static function interpolate($message, array $context = array())
```

Comment:
I guess if there is no use of `$this` AND the function is pure, then I don't have a reason for a method not to be static. Since these two things are true, then I rescind my comment.